scaleway/helpers.go

```diff
@@ -117,3 +126,27 @@ func waitForServerState(scaleway *api.API, serverID, targetState string) error {
 	_, err := stateConf.WaitForState()
 	return err
 }
+
+func retry(f func() error) error {
+	return retryWithCodes([]int{429}, f)
+}
+
+func retryWithCodes(retryCodes []int,
+	f func() error) error {
+	return resource.Retry(serverWaitTimeout, func() *resource.RetryError {
```

Comment:
Please introduce a new constant for this - `serverWaitTimeout` is specific to the boot time of scaleway servers.